functions.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
```

Comment:
i know you haven't done documentation yet but note for future to do docblock thing

Reply:
Got testing then documenting is last :)